client/js/renderPreview.js

```diff
@@ -61,3 +61,44 @@ function handleImageInPreview(content, container) {
 		});
 	}
 }
+
+/* Image viewer */
+
+// FIXME Remove #input focus when this is open
+$("#viewport").on("click", ".toggle-thumbnail", function() {
+	const link = $(this);
+
+	openImageViewer(link);
+
+	// Prevent the link to open a new page since we're opening the image viewer,
+	// but keep it a link to allow for Ctrl/Cmd+click
+	return false;
+});
+
+$(document).on("click", "#image-viewer", function() {
```

Comment:
Pretty sure the way you're binding the event here does not help and `$("#image-viewer").on("click")` should work exactly the same.